sdk/textanalytics/ai-text-analytics/test/public/utils/recordedClient.ts

```diff
@@ -14,7 +14,7 @@ import "./env";
 const replaceableVariables: { [k: string]: string } = {
   AZURE_CLIENT_ID: "azure_client_id",
   AZURE_CLIENT_SECRET: "azure_client_secret",
-  AZURE_TENANT_ID: "azure_tenant_id",
+  AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
```

Comment:
was this intentional?

Reply:
JS Identity package added a new validation for the "syntax" of the tenant-ids before making the requests.
This id(which is a fake value) is supposed to pass through that validation in playback.